### PR TITLE
[UI] Fixes for PlaceCard misalignment issue and Tags overflow [DG] Updates to DG from Sections 1 to 3.1.1

### PR DESCRIPTION
**Pull request by shaungohsc**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -74,9 +79,9 @@ Optionally, you can follow the <<UsingCheckstyle#, UsingCheckstyle.adoc>> docume
 
 ==== Updating documentation to match your fork
 
-After forking the repo, the documentation will still have the SE-EDU branding and refer to the `se-edu/addressbook-level4` repo.
+After forking the repo, the documentation will still have the SE-EDU branding and refer to the `se-edu/TravelBuddy-level4` repo.
 
-If you plan to develop this fork as a separate product (i.e. instead of contributing to `se-edu/addressbook-level4`), you should do the following:
+If you plan to develop this fork as a separate product (i.e. instead of contributing to `se-edu/TravelBuddy-level4`), you should do the following:
```
Comment (PremBagda) [inline docs/DeveloperGuide.adoc]: Perhaps the section on "updating documentation to match your fork" can be removed as it is not relevant to TravelBuddy. Do note that "se-edu/TravelBuddy-level4" does not exist.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -253,72 +264,77 @@ image::StorageClassDiagram.png[width="800"]
...
 
 * can save `UserPref` objects in json format and read it back.
-* can save the Address Book data in json format and read it back.
+* can save the TravelBuddy data in json format and read it back.
 
 [[Design-Commons]]
 === Common classes
 
-Classes used by multiple components are in the `seedu.addressbook.commons` package.
+Classes used by multiple components are in the `seedu.TravelBuddy.commons` package.
```
Comment (PremBagda) [inline docs/DeveloperGuide.adoc]: There is a typo here. The correct name is `seedu.travel.commons`.

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -253,72 +264,77 @@ image::StorageClassDiagram.png[width="800"]
...
+Called by the `undo` and `redo` commands to undo and redo changes to TravelBuddy Place entries.
+Undoable commands include `add` `edit` `delete` `deletem` `clear` `generate` and 'redo'
+Redoable commands include `add` `edit` `delete` `deletem` `clear` `generate` and 'undo'
+
 ==== Current Implementation
 
-The undo/redo mechanism is facilitated by `VersionedAddressBook`.
-It extends `AddressBook` with an undo/redo history, stored internally as an `addressBookStateList` and `currentStatePointer`.
+The undo/redo mechanism is facilitated by `VersionedTravelBuddy`.
+It extends `TravelBuddy` with an undo/redo history, stored internally as an `TravelBuddyStateList` and `currentStatePointer`.
```
Comment (PremBagda) [inline docs/DeveloperGuide.adoc]: The correct variable name is `travelBuddyStateList`.

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (shaungohsc) [reply docs/DeveloperGuide.adoc]: Removed the section

File: docs/DeveloperGuide.adoc (hunk 3 above)
Comment (shaungohsc) [reply docs/DeveloperGuide.adoc]: Thank you for pointing that out

**Pull request merged**